pants.toml

```diff
@@ -48,7 +49,17 @@ string_imports = true
 lockfile = "lockfiles/pytest.txt"
 version = "pytest>=6.2.4,<6.3"
 extra_requirements.add = [
-  "pytest-django>=3.9.0,<4",
+  "pytest-django>=4,<5",
 ]
 
 execution_slot_var = "PANTS_EXECUTION_SLOT"
+
+[mypy]
+lockfile = "lockfiles/mypy.txt"
+version = "mypy==0.942"
+interpreter_constraints = [">=3.7"]
+extra_requirements.add = [
+  # This is a mypy plugin, as well as a type stubs repository. So it must be mentioned
+  # here as well as in requirements.txt.
```

Comment:
Did you use it in requirements.txt rather than `[mypy].extra_type_stubs` so that it's in a lockfile?

Reply:
Per our 1:1, feel free to open an issue about how `[mypy].extra_type_stubs` is not locked down -- I agree with you that we want to fix that, it's crucial Pants helps you mitigate supply chain attacks.

Reply:
> Did you use it in requirements.txt rather than `[mypy].extra_type_stubs` so that it's in a lockfile?

That and for consistency with types-requests, which seemed less sensible to put in extra_type_stubs, since we only use requests in one place in the repo.